public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Authors Haven</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="main.css">
+  <script src="main.js"></script>
```

Comment:
@adaezeodurukwe is there a reason you have a script tag pointing to main.js

Reply:
I created a standard HTML5 boilerplate to be edited subsequently to suit the project need but I see how it is unnecessary now, I'll fix this. Thanks.